### PR TITLE
Add assembly scene

### PR DESCRIPTION
**Pull request by ajd6735**
Adds a new assembly scene and modifies the order of the assembly scenes

### CODE REVIEW

**Review by zlix12 — DISMISSED** [review]
Looks great

**Review by AvinaashGhansam — DISMISSED** [review]

File: src/scene/get-scene-config.ts [hunk 1]
```diff
@@ -25,15 +26,27 @@ const defaultCameraPosition = new Vector3(0, 6, 25);
...
+        audio: 'sounds/assemble.wav',
+        buttonText: 'Assemble Orbiter'
+      }
+    },
     [SceneName.VIBRATION_TESTING]: {
       component: VibrationTestingScene,
       markerUrl: 'assets/patt.hiro',
+      previousSceneTransition: {
+        toScene: SceneName.ASSEMBLY,
+        // animation: AnimationName.BACK,
```
Comment (zlix12) [inline src/scene/get-scene-config.ts:47]: Should this be un-commented?

**Review by zlix12 — CHANGES_REQUESTED** [review]
Looks good just one small thing noted

**Review by zlix12 — APPROVED** [review]
LGTM

**Pull request merged**